doc/source/faq.rst

```diff
@@ -0,0 +1,76 @@
+.. _faq:
+
+==========================
+Frequently Asked Questions
+==========================
+
+.. contents:: Contents
+   :local:
+
+
+Inverse Solution
+================
+
+How should I regularize the covariance matrix
+---------------------------------------------
+
+The MNE manual suggests regularizing the noise covariance matrix (see
+:ref:`CBBHEGAB`). In MNE-Python, this is done using the regularization methods
+described in [1, 2, 3, 4]. For this the 'auto' option can be used. With this
```

Comment:
And is this the proper way to make references to the articles below? I thought you'd need some `_`'s in there.